source/tests/pt/model/test_atomic_model_atomic_stat.py

```diff
@@ -212,6 +212,8 @@ def cvt_ret(x):
             self.merged_output_stat, stat_file_path=self.stat_file_path
         )
         ret1 = md0.forward_common_atomic(*args)
+        expected_std = np.ones((2, 2, 2))  # 2 keys, 2 atypes, 2 max dims.
```

Comment:
shouldn't the std of type 0  be 0? All atomic labels of type 0 are 5.0.

Reply:
> shouldn't the std of type 0 be 0? All atomic labels of type 0 are 5.0.

the std are updated to 1 in `_post_process_stat`, if we want the actual input, we would need to turn that off.

Reply:
the `_post_process_stat` should only be applied to global stat, because in that case the std of type is not available. In the case of atomic stat, we use the std from data for each atom type.